docs/CHANGES.rst

```diff
@@ -1,3 +1,32 @@
+v29.0.0 (2025-03-26)
+====================
+
+New Features
+------------
+
+- Added tools for comparing schemas in the `diff` module.
+  These can be run from the command line with `felis diff`. (`DM-46130 <https://jira.lsstcorp.org/browse/DM-46130>`_)
+- Added column grouping functionality to the data model.
+  Column groups are sets of related columns that, in addition to the standard object attributes, may have an `ivoa:ucd`.
+  Information on column groups was also added to the User Guide. (`DM-48427 <https://jira.lsstcorp.org/browse/DM-48427>`_)
+
+
+Other Changes and Additions
+---------------------------
+
+- Added Python 3.13 to supported versions.
+  No code changes were required. (`DM-47900 <https://jira.lsstcorp.org/browse/DM-47900>`_)
+
+
+An API Removal or Deprecation
+-----------------------------
+
+- Removed the ``apply_schema_to_tables`` flag on the ``MetaDataBuilder`` which creates SQLAlchemy ``MetaData`` from a Felis ``Schema``.
+  This was a redundant variable as enabling the ``apply_schema_to_metadata`` flag already correctly associated the metadata to the schema. (`DM-47256 <https://jira.lsstcorp.org/browse/DM-47256>`_)
+- Removed the `tap` module, which was replaced by `tap_schema`.
+  The `init-tap` command was removed from the CLI and replaced with `init-tap-schema`. (`DM-48616 <https://jira.lsstcorp.org/browse/DM-48616>`_)
+
+
```

Comment:
The section heading below has to be deleted.

Reply:
In other packages we put the section heading in the rst that is embedding this so that towncrier can keep prepending to this file.

Reply:
Good catch - I have "Changelog" as a header for site organization, so I will move it to the top.

Reply:
As discussed on Slack, I'll instead remove the header from the page and update the site index.